actions/release-candidate/README.md

```diff
@@ -167,16 +167,10 @@ Perform the following steps to test changes to the daffodil-infrastructure repo
    `.github/workflows/release-candidate.yml` file of the repository to be tested on. Then, 
     push your changes to your fork of the test repository. 
 2. Add the secrets required by the `ASF Release Candidate` step to your
-    test repository. The following secrets are required:
+    test repository. The following secret is required:
 
    - DAFFODIL_GPG_SECRET_KEY (any private key without a passphrase)
```

Comment:
What are your thoughts on making `DAFFODIL_GPG_SECRET_KEY` also optional if not publishing? It would make testing this so much easier if we don't require any secrets at all, especial gpg keys which are kindof annoying to generate/export/etc.

Maybe if publishing is disabled then we don't require the input. We use the value if it's not empty, but if it is empty then we generate a GPG key public+secret pair and sign with the secret. We could also extract and upload the public key as another artifact when publishing is disabled in post.js so that users could download and import the key if they wanted to verify signatures.